source/clear-linux/guides/network/network-bonding.rst

```diff
@@ -60,21 +66,20 @@ __ https://www.freedesktop.org/software/systemd/man/systemd.netdev.html#id-1.20.
       [Link]
       MTUBytes=9000
 
-   This example demonstrates bonding all four ports of a quad-port NIC, with
-   names in the range ``enp1s0f0-enp1s0f3``, allowing the use of a single file
-   with a wildcard match. You may also create a separate file for each NIC,
-   particularly if they have names that are not wildcard-friendly. This
-   configuration assigns each NIC as a slave of ``bond1``. For best results,
-   do not assign addresses or DHCP support to the individual NICs.
+   The example bonds all four ports of a quad-port NIC as a slave of `bond1`.
+   The example uses a wildcard match because the NIC names are in the range
+   `enp1s0f0-enp1s0f3`. If your NIC names are not wildcard-compatible, create
+   a separate :file:`.network` file for each NIC.
+
+   For best results, do not assign addresses or DHCP support to the individual
+   NICs.
 
-   This example also enables jumbo frames of up to 9000 bytes to optimize large
-   data transfers on the local network. Again, your NICs and switch must
-   support jumbo frames, and your switch may require additional configuration.
+   The `[Link]` setting enables jumbo frames of up to 9000 bytes. Your switch
```

Comment:
[Link] is just the section name. MTUBytes is the setting whose value determines whether jumbo frames are supported.

Reply:
Fixed.